openedx/core/djangoapps/credit/api.py

```diff
@@ -263,6 +267,8 @@ def create_credit_request(course_key, provider_id, username):
 
     if created:
         credit_request.uuid = uuid.uuid4().hex
+    else:
+        credit_request.timestamp = datetime.datetime.now(pytz.UTC)
```

Comment:
why not (`.auto_now`)[https://docs.djangoproject.com/en/1.4/ref/models/fields/#django.db.models.DateField.auto_now] - does that not work with timezone-awareness?